tests/test_slogpy_console.py

```diff
@@ -0,0 +1,50 @@
+"""The test for the slogpy package."""
+from slogpy import slog
+from slogpy import __version__
+from slogpy.slog import Slog as slog
+def test_version():
```

Comment:
This should probably be in a separate file since it's not testing console output